### ISSUE TITLE
Flexbox and custom-selectors

### ISSUE THREAD

**Issue by sunny-g** [postcss/autoprefixer#730]
I'm having a hard-to-reproduce issue with this plugin in conjunction with a few others.

I have a file for overriding some globals from a third-party dependency, and to make it easier to override my overrides, I created my own custom-selectors, e.g.:

```css
@custom-selector :--Select :global(.Select-control);
:--Select {
    border-radius: 2px;
    border: none;
    height: auto;

    /* the dropdown menu */
    &__dropdown {
        &__menu {
            background-color: background--dark;
            border: none;
        }
    }
    /* ... more styles here */
}
```

and elsewhere in my components, I am writing CSS as such:

```css
.imageryPreview {
    @mixin flex-col space-between, center;

    & :--Select {
        align-self: flex-start;  /* this is the line causing the error */
        background-color: input__button--secondary;
        width: 206px;
    }
}
```

Everytime I try to write styles that can be autoprefixed (like flexbox properties) that use these custom-selectors, I get the following error:

```sh
ERROR in ./~/css-loader?modules&sourceMap&importLoaders=1&localIdentName=[local]___[hash:base64:5]!./~/postcss-loader!./widgets/static/widgets/ImageryAnalyticsPreviewPanel/ImageryAnalyticsPreviewPanel.styles.css
Module build failed: TypeError: Cannot read property '1' of null
    at AlignSelf.Prefixer.parentPrefix (/home/sunnyg/gitdev/descarteslabs/widgets/node_modules/autoprefixer/lib/prefixer.js:79:275)
    at AlignSelf.Prefixer.parentPrefix (/home/sunnyg/gitdev/descarteslabs/widgets/node_modules/autoprefixer/lib/prefixer.js:79:362)
    at AlignSelf.Prefixer.process (/home/sunnyg/gitdev/descarteslabs/widgets/node_modules/autoprefixer/lib/prefixer.js:91:21)
    at AlignSelf.Declaration.process (/home/sunnyg/gitdev/descarteslabs/widgets/node_modules/autoprefixer/lib/declaration.js:144:50)
    at /home/sunnyg/gitdev/descarteslabs/widgets/node_modules/autoprefixer/lib/processor.js:111:26
    at /home/sunnyg/gitdev/descarteslabs/widgets/node_modules/postcss/lib/container.js:192:28
    at /home/sunnyg/gitdev/descarteslabs/widgets/node_modules/postcss/lib/container.js:148:26
    at Rule.each (/home/sunnyg/gitdev/descarteslabs/widgets/node_modules/postcss/lib/container.js:114:22)
    at Rule.walk (/home/sunnyg/gitdev/descarteslabs/widgets/node_modules/postcss/lib/container.js:147:21)
    at /home/sunnyg/gitdev/descarteslabs/widgets/node_modules/postcss/lib/container.js:150:32
 @ ./widgets/static/widgets/ImageryAnalyticsPreviewPanel/ImageryAnalyticsPreviewPanel.styles.css 4:14-241
```

Any ideas?

**Comment by ai** [comment]
I will look it in Friday. Feel free to ping me in Sunday if I will forget.

**Comment by ai** [comment]
Fixed 8f7e0e6

**Comment by ai** [comment]
Released in 6.5.1

**Comment by sunny-g** [comment]
@ai Thank you!!
